jib-gradle-plugin/CHANGELOG.md

```diff
@@ -9,6 +9,16 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+## 0.9.8
+
+### Added
+
+- Docker context generation now includes snapshot dependencies and extra files ([#516](https://github.com/GoogleContainerTools/jib/pull/516/files))
+
+### Changed
+
+- Only builds non-empty layers ([#516](https://github.com/GoogleContainerTools/jib/pull/516/files))
```

Comment:
Is it true? I saw above

```java
   // TODO: Don't generate empty layers.
   /**
    * Creates the Docker context in {@code #targetDirectory}.
```

Reply:
That's just for generating the Docker context; I don't think empty layers are added to the image itself in `jib:build` etc.

Reply:
Oops forgot to remove that - it actually skips the empty layers now.